your-first-cluster.md

```diff
@@ -44,6 +46,9 @@ In this case we're using the Docker infrastructure provider - so we need a the D
 Cluster API's CLI - `clusterctl` is used to install the CAPI Management Cluster components - luckily it's able to handle installing all of the above as well as the Custom Resource Definitions used by the Cluster API controllers.
 
 ```bash
+# Please ensure CLUSTERCTL_REPOSITORY_PATH is set and points to the clusterctl repository. 
+export CLUSTERCTL_REPOSITORY_PATH=$(pwd)/clusterctl/repository
```

Comment:
Just as a reminder. I'm not 100% sure folks necessarily have this still set in the current shell from prereqs (especially if they did prereqs before)